### ISSUE TITLE
ssd_mobilenet_edgetpu_cocoalways reports 50% score for positive classifications

### ISSUE THREAD

**Issue by NicholaiStaalung** [tensorflow/models#8469]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
Debian 9.9 (Google Cloud Platform)
- Mobile device (e.g., Pixel 4, Samsung Galaxy 10) if the issue happens on mobile device:
no
- TensorFlow installed from (source or binary):
Binary
- TensorFlow version (use command below):
1.14
- Python version:
3.5.3
- Bazel version (if compiling from source):
0.28.0
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
10
- GPU model and memory:
P100 (15gb)

**Please provide the entire URL of the model you are using?**

**Describe the current behavior**

**Describe the expected behavior**

**Code to reproduce the issue**

**Other info / logs**

**Comment by pkulzc** [comment]
I didn't understand your question. Are you saying "when finetuning from that checkpoint the mAP for your training job look weird?"

**Comment by NicholaiStaalung** [comment]
Im saying that when i walk through individual predictions they are always scored as 50%. Which is odd in two ways
1. 50% should result in negative 
2 Model doesn't differentiate between individual classes or detection's

Maybe it is something with the activation function for the output. I have played with both SIGMOID and SOFTMAX. However it is kind of hard for me to A/B test. 

I have recently trained new models where predictions are returned higher than 50%. But they are still for most returned as the same value across observations. I think its weird and not plausible.

**Comment by fossilM** [comment]
I've been dealing with the same issue - what is the design choice here?
